src/ASM/ASMbase.h

```diff
@@ -183,9 +183,13 @@ class ASMbase
   //! \brief Returns the global node number for the given node.
   //! \param[in] inod 1-based node index local to current patch
   virtual int getNodeID(size_t inod, bool = false) const;
+  //! \brief Returns local 1-based index of the element with given global number.
+  //! \details If the given node number is not present, 0 is returned.
+  //! \param[in] globalNum Global node number
+  virtual size_t getElmIndex(int globalNum) const;
```

Comment:
akva: "Make the function virtual"
kmo: "Remove the virtual"
viking: [I have no strong feelings one way or the other](https://youtu.be/ussCHoQttyQ)

I've removed the virtual for now and we'll revisit if we need it at a later time.